docs/lib/storage/fragments/ios/getting-started.md

```diff
@@ -61,22 +57,25 @@ If this is a new project, run `pod init` from the root of your application folde
 ```ruby
 target :'YOUR-APP-NAME' do
 use_frameworks!
-    pod 'AmplifyPlugins/AWSS3StoragePlugin'
-    pod 'AWSMobileClient', '~> 2.13.0'
+  pod 'AmplifyPlugins/AWSS3StoragePlugin'
+  pod 'AWSMobileClient', '~> 2.13.0'
```

Comment:
i think the new docs have removed all versions for the podFile, ie in Analytics
```
target :'YOUR-APP-NAME' do
  use_frameworks!
  pod 'AmplifyPlugins/AWSPinpointAnalyticsPlugin'
  pod 'AWSMobileClient'
end
```

Reply:
done!